### PR TITLE
Speed Limit Watcher example with tutorial for developer.here.com

### PR DESCRIPTION
**Pull request by mkarpicki**

### CODE REVIEW

File: speed-limit-watcher/.gitignore [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.gradle/
+build/
+app/libs/
+!app/libs/README.md
+.idea/
+*.iml
+local.properties
```
Comment (sschuberth) [inline speed-limit-watcher/.gitignore]: Nit: Please add a final newline.

File: speed-limit-watcher/app/build.gradle [hunk 2]
```diff
@@ -0,0 +1,36 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 28
+    defaultConfig {
+        applicationId "com.here.example.speedlimit"
```
Comment (sschuberth) [inline speed-limit-watcher/app/build.gradle]: Nit: Prefer single quotes where no variable interpolation is needed.

File: speed-limit-watcher/app/build.gradle [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    }
+    repositories {
+        flatDir {
+            dirs 'libs'
+        }
+    }
+}
+
+dependencies {
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
```
Comment (sschuberth) [inline speed-limit-watcher/app/build.gradle]: Is `*.jar` really needed? If so, still consider merging this with the patterns in the line below.

File: speed-limit-watcher/app/build.gradle [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    implementation fileTree(dir: 'libs', include: ['*.aar'])
+    implementation 'com.android.support:appcompat-v7:28.0.0-rc01'
+    implementation 'com.android.support.constraint:constraint-layout:1.1.2'
+    implementation 'com.android.support:support-v4:28.0.0-rc01'
+    testImplementation 'junit:junit:4.12'
+    androidTestImplementation 'com.android.support.test:runner:1.0.2'
+    androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.2'
+    //compile(name: 'HERE-sdk', ext: 'aar')HERE-sdk.aar
```
Comment (sschuberth) [inline speed-limit-watcher/app/build.gradle]: Please do not commit out-commented lines.

File: speed-limit-watcher/.gitignore (hunk 1 above)
Comment (sschuberth) [reply speed-limit-watcher/.gitignore]: Likewise in several `*.xml` files below.

File: speed-limit-watcher/build.gradle [hunk 5]
```diff
@@ -0,0 +1,27 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+
+buildscript {
+    
+    repositories {
+        google()
+        jcenter()
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:3.1.4'
```
Comment (sschuberth) [inline speed-limit-watcher/build.gradle]: Please [align](https://travis-ci.org/heremaps/here-android-sdk-examples/builds/450641636#L1399) on version 3.2.1

File: speed-limit-watcher/app/src/main/AndroidManifest.xml [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+        android:label="@string/app_name"
+        android:roundIcon="@mipmap/ic_launcher_round"
+        android:supportsRtl="true"
+        android:theme="@style/AppTheme">
+
+        <meta-data android:name="com.here.android.maps.appid" android:value="{SAMPLE_APP_ID}"/>
+        <meta-data android:name="com.here.android.maps.apptoken" android:value="{SAMPLE_APP_CODE}"/>
+        <meta-data android:name="com.here.android.maps.license.key" android:value="{SAMPLE_LICENSE}"/>
+
+        <!--<service-->
```
Comment (SirBif) [inline speed-limit-watcher/app/src/main/AndroidManifest.xml]: Can you remove this commented out block?

File: speed-limit-watcher/.gitignore [hunk 7]
```diff
@@ -0,0 +1,7 @@
+.gradle/
+build/
+app/libs/
```
Comment (SirBif) [inline speed-limit-watcher/.gitignore]: you have 2 commits touching this file. can you squash them?

**Comment by sschuberth** [comment]
Superseded by https://github.com/heremaps/here-android-sdk-examples/pull/177.

**Pull request closed without merging**